wp-post-meta-revisions.php

```diff
@@ -134,6 +134,10 @@ public function _wp_save_revisioned_meta_fields( $revision_id ) {
 		foreach ( $this->_wp_post_revision_meta_keys() as $meta_key ) {
 			$meta_value = get_post_meta( $post_id, $meta_key );
 
+			if ( empty( $meta_value ) ) {
```

Comment:
@dan-westall yep, because `$single` (3rd param to `get_post_meta`) is false here, it's always an array, so `empty()` is a fine check to use for non-existent meta.